src/app/modules/main/module.nim

```diff
@@ -344,6 +344,29 @@ proc createChannelGroupItem[T](self: Module[T], channelGroup: ChannelGroupDto):
     chatMember.role = MemberRole.None
     members.add(chatMember)
 
+
+  var bannedMembers = newSeq[MemberItem]()
+  for memberId, memberState in communityDetails.pendingAndBannedMembers.pairs:
+    let state = memberState.toMembershipRequestState()
+    if state == MembershipRequestState.Banned or state == MembershipRequestState.UnbannedPending:
+      let contactDetails = self.controller.getContactDetails(memberId)
+      let memberItem = initMemberItem(
+        pubKey = memberId,
+        displayName = contactDetails.dto.displayName,
+        ensName = contactDetails.dto.name,
+        isEnsVerified = contactDetails.dto.ensVerified,
+        localNickname = contactDetails.dto.localNickname,
+        alias = contactDetails.dto.alias,
+        icon = contactDetails.icon,
+        colorId = contactDetails.colorId,
+        colorHash = contactDetails.colorHash,
+        onlineStatus = toOnlineStatus(self.controller.getStatusForContactWithId(memberId).statusType),
+        isContact = contactDetails.dto.isContact,
+        isVerified = contactDetails.dto.isContactVerified(),
+        membershipRequestState = state,
+      )
```

Comment:
Small suggestion, maybe we can create util function in this module where we pass the `contactDetails` and `memberId` and it does the `initMemberItem`.  That way we don't have to duplicate all those lines.

Reply:
done 👍